examples/core/Graphormer/train.py

```diff
@@ -0,0 +1,493 @@
+        num_workers=16,
+    )
+
+    # load pretrained model
+    download(url="https://data.dgl.ai/pre_trained/graphormer_pcqm.pth")
+    model = Graphormer()
+    state_dict = th.load("graphormer_pcqm.pth")
+    model.load_state_dict(state_dict)
+    # reset output layer parameters
+    model.reset_output_layer_parameters()
+    num_epochs = 16
+    tot_updates = 33000 * num_epochs / params.batch_size
+    warmup_updates = tot_updates * 0.16
+
+    optimizer = AdamW(model.parameters(), lr=1e-4, eps=1e-8, weight_decay=0)
+    lr_scheduler = get_polynomial_decay_schedule_with_warmup(
+        optimizer,
+        num_warmup_steps=warmup_updates,
+        num_training_steps=tot_updates,
+        lr_end=1e-9,
+        power=1.0,
+    )
+
+    epoch_train_AUCs, epoch_val_AUCs, epoch_test_AUCs = [], [], []
+
+    # multi-GPUs
+    (
+        model,
+        optimizer,
+        train_loader,
+        val_loader,
+        test_loader,
+        lr_scheduler,
+    ) = accelerator.prepare(
+        model, optimizer, train_loader, val_loader, test_loader, lr_scheduler
+    )
+
+    for epoch in range(num_epochs):
+
+        epoch_train_loss, epoch_train_auc = train_epoch(
+            model, optimizer, train_loader, lr_scheduler
+        )
+        epoch_val_loss, epoch_val_auc = evaluate_network(model, val_loader)
+        epoch_test_loss, epoch_test_auc = evaluate_network(model, test_loader)
+
+        epoch_train_AUCs.append(epoch_train_auc)
+        epoch_val_AUCs.append(epoch_val_auc)
+        epoch_test_AUCs.append(epoch_test_auc)
+
+        accelerator.print(
+            "Epoch={}, lr={}, train_ROCAUC={:.3f}, val_ROCAUC={:.3f}, test_ROCAUC={:.3f}".format(
+                epoch + 1,
+                optimizer.param_groups[0]["lr"],
+                epoch_train_auc,
+                epoch_val_auc,
+                epoch_test_auc,
+            )
+        )
+
+    # Return test and train metrics at best val metric
+    index = epoch_val_AUCs.index(max(epoch_val_AUCs))
+    val_auc = epoch_val_AUCs[index]
+    train_auc = epoch_train_AUCs[index]
+    test_auc = epoch_test_AUCs[index]
+
+    accelerator.print("Test ROCAUC: {:.4f}".format(test_auc))
+    accelerator.print("Val ROCAUC: {:.4f}".format(val_auc))
+    accelerator.print("Train ROCAUC: {:.4f}".format(train_auc))
+    accelerator.print("Best epoch index: {:.4f}".format(index))
+
+
+class MolHIVDataset(th.utils.data.Dataset):
+    def __init__(self):
+
+        dataset = DglGraphPropPredDataset(name="ogbg-molhiv")
+        split_idx = dataset.get_idx_split()
+
+        # shortest path distance
+        for g, label in dataset:
+            spd, path = shortest_dist(g, root=None, return_paths=True)
+            g.ndata["spd"] = spd
+            g.ndata["path"] = path
+
+        self.train, self.val, self.test = (
+            dataset[split_idx["train"]],
+            dataset[split_idx["valid"]],
+            dataset[split_idx["test"]],
+        )
+
+        accelerator.print(
```

Comment:
why accelerator.print?

Reply:
It can avoid repeating output when using multiple GPUs.